docs/src/docs/core/Overlay.mdx

```diff
@@ -28,6 +28,12 @@ You can change overlay opacity (from 0 to 1), color (CSS color value, not connec
 
 <Demo data={OverlayDemos.gradient} />
 
+## With blur
+
+To blur the overlay backdrop, set the `blur` prop with a number. This will set the backdrop blur in px.
```

Comment:
Should [browser support](https://developer.mozilla.org/en-US/docs/Web/CSS/backdrop-filter#browser_compatibility) be mentioned here?

Reply:
Nope it's fine as is